docs/user-documentation/metadata.md

```diff
@@ -11,27 +11,27 @@ As described in the [resource nodes section](resource-nodes.md), Islandora 8 dig
+Not all Content Types in your Drupal site need be [*Islandora Resource Nodes*](resource-nodes.md). Making a Content Type a Resource Node will associate Islandora specific behaviours (such as syncing to Fedora or causing derivatives to be generated) with it. The decision to make a content an Islandora resource node is left to the discretion of the site manager. In Islandora, a "resource node" is usually considered a descriptive record for "a thing", and is conceptually similar to an "Islandora Object" in 7.x, i.e. a "Fedora Object" in Fedora 3.x and below. [Read more on configuring a content type to be treated as a Resource Node](content_types.md#create-a-content-type).
 
 ### Fields
 
-The administrator will define the fields that are associated with a specific Content Type. The same fields can be applied to different Content Types, but the field display and editing configurations are unique to each Content Type. The names and definitions of these fields are specific to Drupal and do not have to correspond to an outside metadata schema. You will give each field a Label, Machine Name, and a specific [Field Type](#fields), like Text, Integer, EDTF, or Entity Reference (see below). Specific to the Field Type you will then define the maximum length of the field, the number of values it can contain, and what taxonomies it might link to.
+The administrator will define the fields that are associated with a specific Content Type. The same fields can be applied to different Content Types, but the field display and editing configurations are unique to each Content Type. The names and definitions of these fields are specific to Drupal and do not have to correspond to an outside metadata schema. You will give each field a Label, Machine Name, and a specific [Field Type](#field-types), like Text, Integer, EDTF, or Entity Reference (see below). Specific to the Field Type you will then define the maximum length of the field, the number of values it can contain, and what taxonomies it might link to.
 
-Fields can be added under Content Type \> Manage Fields. This tab will list all Fields, their Label, Machine Name, Field Type, and give you the option to make what edits to the definition of that field that you can.
+Fields can be added under **Administration** >> **Structure** >> **Content types** >> _Your Content Type's Name_ >> **Manage fields** (/admin/structure/types/your_type/fields). This tab will list all Fields, their Label, Machine Name, Field Type, and give you the option to make what edits to the definition of that field that you can.
```

Comment:
Made this formatting follow [Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/). 

Except it is weird, because the exact path to follow (and url path) depends on what content type you want to manage fields in. Subsequent examples all use the islandora_defaults Repository Item, so maybe we want a note at the top that examples are all referring to that, and if you have created your own content type(s), the navigation path to manage those will be slightly different?